libs/expo/betterangels/src/lib/screens/Client/Profile/DemographicInfo.tsx

```diff
@@ -0,0 +1,130 @@
+import { StyleSheet, View } from 'react-native';
+import {
+  enumDisplayEyeColor,
+  enumDisplayHairColor,
+  enumDisplayMaritalStatus,
+  enumDisplayRace,
+} from '../../../static/enumDisplayMapping';
+import { IProfileSectionProps } from './types';
+
+const InfoRow = ({
+  label,
+  value,
+}: {
+  label: string;
+  value?: string | null;
+}) => (
+  <View style={styles.flexRow}>
+    <TextRegular size="sm">{label}</TextRegular>
+    <TextBold size="sm">{value || 'N/A'}</TextBold>
+  </View>
+);
+
+export default function DemographicInfo(props: IProfileSectionProps) {
+  const { expanded, setExpanded, client } = props;
+
+  const isDemographicInfo = expanded === 'Demographic Info';
+
+  function convertToFeetAndInches(inches: number | null | undefined): string {
+    const feet = Math.floor((inches || 0) / 12);
+    const remainingInches = (inches || 0) % 12;
+    return `${feet}'${remainingInches}"`;
+  }
+
+  return (
+    <Accordion
+      expanded={expanded}
+      setExpanded={() => {
+        setExpanded(isDemographicInfo ? null : 'Demographic Info');
+      }}
+      mb="xs"
+      title="Demographic Info"
+    >
+      {isDemographicInfo && (
+        <View
+          style={{
+            height: isDemographicInfo ? 'auto' : 0,
+            overflow: 'hidden',
+          }}
+        >
+          <CardWrapper>
+            <View style={{ gap: Spacings.lg }}>
+              <InfoRow
+                label="Gender"
+                value={client?.clientProfile.displayGender}
+              />
+              <InfoRow
+                label="Pronouns"
+                value={client?.clientProfile.displayPronouns}
+              />
+              <InfoRow
+                label="Race"
+                value={
+                  client?.clientProfile.race &&
+                  enumDisplayRace[client?.clientProfile.race]
+                }
+              />
+              <InfoRow
+                label="City of Birth"
+                value={client?.clientProfile.placeOfBirth}
+              />
+              <InfoRow
+                label="Height"
+                value={
+                  client?.clientProfile.heightInInches
+                    ? convertToFeetAndInches(
+                        client?.clientProfile.heightInInches
+                      )
+                    : ''
+                }
+              />
+              <InfoRow
+                label="Eye Color"
+                value={
+                  client?.clientProfile.eyeColor &&
+                  enumDisplayEyeColor[client?.clientProfile.eyeColor]
+                }
+              />
+              <InfoRow
+                label="Hair Color"
+                value={
+                  client?.clientProfile.hairColor &&
+                  enumDisplayHairColor[client?.clientProfile.hairColor]
+                }
+              />
+              <View>
+                <TextRegular size="xs">Physical Description</TextRegular>
```

Comment:
Hey @mikefeldberg Thanks for noticing it, however, I think it looks odd right now because there is not space in between the title "Physical Description" and the content that is in bold. Otherwise, that's how it was designed. 

@Davit-BetterAngels ,
Can you give 8px in the between? (I gave my approval earlier but I missed this part!) 

<img width="360" src="https://github.com/user-attachments/assets/99430ea5-e6ef-4923-93aa-785a69fae761" />

Reply:
done

Reply:
Hey @Davit-BetterAngels 

In the view only mode, Physical Description comes after the Martial Status now. I think it should be the other way around. Can you move it back to the way it was?